web/src/frontend/js/sider/LandingssideSide.js

```diff
@@ -78,9 +83,9 @@ export class Container extends Component {
             {
                 (() => {
                     if (henter) {
-                        return <AppSpinner />;
+                        return <AppSpinner/>;
                     } else if (hentingFeilet) {
-                        return <Feilmelding />;
+                        return <Feilmelding/>;
```

Comment:
Er det riktig at vi skal feile hele siden om vi mangler metadata?

Reply:
Hm, godt spørsmål 🤔 det betyr jo mest sansynlig at det er noe feil med syforest da, eller noen baksystemer så da burde andre ting feile også. VI trenger det jo egenltig bare for å vite om vi skal vise lenken så det er kanskje ikke så stress? Jeg bare herma litt ukritisk, tenkte det bare kunne håndteres som alt annet. Hva sier du @helgefredheim ?